src/openforms/emails/digest.py

```diff
@@ -492,3 +523,130 @@ def _report(var_key: str = "", exception: bool = False, rule_index: int = 0):
             )
 
     return invalid_logic_rules
+
+
+def collect_expired_or_near_expiry_referentielijsten_data():
+    config = GlobalConfiguration.get_solo()
+
+    service_tabels_items_mapping = {}
+    for referentielijsten_service in config.referentielijsten_services.all():
+        with build_client(
+            referentielijsten_service, client_factory=ReferentielijstenClient
+        ) as client:
+            try:
+                tabels = client.get_tabellen()
+            except RequestException:
+                tabels = []
+
+            service_label = referentielijsten_service.label
```

Comment:
why are we checking tables and items that may not even be used in any form?

We should only check the form definitions of (active) forms that make use of reference lists, check only those services and tables and detect if there's stuff (about to) expire(d). The reference can have expired tables, but as long as they aren't used, everything is fine.